src/constants.jl

```diff
@@ -0,0 +1,46 @@
+# NOTES:
+#
+# python function manager maintains a local table of "execution info" with a
+# "function_id" key and values that are named tuples of name/function/max_calls.
+#
+# python remote function sets a UUID4 at construction time:
+# https://github.com/beacon-biosignals/ray/blob/7ad1f47a9c849abf00ca3e8afc7c3c6ee54cda43/python/ray/remote_function.py#L128
+#
+# ...that's used to set the function_hash (???)...
+# https://github.com/beacon-biosignals/ray/blob/7ad1f47a9c849abf00ca3e8afc7c3c6ee54cda43/python/ray/remote_function.py#L263-L265
+#
+# later comment suggests that "ideally" they'd use the hash of the pickled
+# function:
+# https://github.com/beacon-biosignals/ray/blob/7ad1f47a9c849abf00ca3e8afc7c3c6ee54cda43/python/ray/includes/function_descriptor.pxi#L183-L186
+#
+# ...but that it's not stable for some reason.  but.....neither is a random
+# UUID?????
+#
+# the function table key is built like
+# <key type>:<jobid>:key
+
+# function manager holds:
+# local cache of functions (keyed by function id/hash from descriptor)
+# gcs client
+# ~~maybe job id?~~ this is managed by the core worker process
```

Comment:
these are pretty specific to the function manager itself, not the constants that got moved here.